tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
 # note that tox expects interpreters to be found at C:\PythonXY,
 # with XY being python version ("27" or "34") for instance
-envlist = py26, py27, py32, py33, py34, docs
+envlist = py{27,34}-pyqt4, py34-pyqt5, py{26,27,32,33,34}-pyside, docs
+
+
 [testenv]
-deps=pytest 
-    pyside    
+deps=pytest
+    py{26,27,32,33,34}-pyside: pyside
 commands=py.test {posargs}
+setenv=
+    py{26,27,32,33,34}-pyside: PYTEST_QT_API = pyside
+    py{26,27,32,33,34}-pyqt4: PYTEST_QT_API = pyqt4
+    py{26,27,32,33,34}-pyqt5: PYTEST_QT_API = pyqt5
+sitepackages=True
+passenv=QT_QPA_PLATFORM_PLUGIN_PATH
```

Comment:
Yep, qutebrowser symlinks (on Linux/OS X) or copies (on Windows) PyQt into the virtualenv as I wanted a clearly isolated environment. Feel free to copy `link_pyqt.py` into your repo and license it as LGPL3 if you want to do the same.